ethereum-financial-v2/src/services/pools-balances/pool-balance.srv.ts

```diff
@@ -0,0 +1,389 @@
+import { Logger } from 'winston'
+import { BlockDto } from '../../entity/events'
+import { elapsedTime } from '../../utils/time'
+import BigNumber from 'bignumber.js'
+import * as E from 'fp-ts/Either'
+import { networkAlert, NetworkError } from '../../utils/errors'
+import { PoolBalanceCache } from './pool-balance.cache'
+import { Finding, FindingSeverity, FindingType } from 'forta-agent'
+import { ETH_DECIMALS } from '../../utils/constants'
+
+//  MIN_5 Math.ceil((5 * 60) / 13) = 23
```

Comment:
```suggestion
//  MIN_5 Math.ceil((5 * 60) / 12) = 25
```

Ethereum PoW had ~13 seconds block time
Ethereum PoS has =12 seconds block time